spec/serializers/v1/commitment_serializer_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ::V1::CommitmentSerializer do
+  subject(:serializer) do
+    described_class.new(commitment, root_name: 'commitment', includes: %i[taxes])
+  end
+
+  let(:commitment) { create(:commitment) }
+  let(:tax) { create(:tax, organization: commitment.plan.organization) }
+  let(:commitment_applied_tax) { create(:commitment_applied_tax, commitment:, tax:) }
+
+  before { commitment_applied_tax }
+
+  it 'serializes the object' do
+    result = JSON.parse(serializer.to_json)
+
+    aggregate_failures do
+      expect(result['commitment']['lago_id']).to eq(commitment.id)
```

Comment:
can we refactor this to have only one expectation?

something like `expect(result['commitment']).to include(...)`